src/main/java/racingcargame/Distance.java

```diff
@@ -0,0 +1,17 @@
+package racingcargame;
+
+public class Distance {
+    private int value = 0;
+
+    public Distance(int value) {
+        this.value = value;
+    }
+
+    public int increase() {
+        return value += 1;
```

Comment:
value증가에 대한 부분은 아래와 같이도 작성할 수 있어보이네요 🙇🏻‍♀️
```suggestion
        return value ++;
```